Formula/luau.rb

```diff
@@ -0,0 +1,22 @@
+class Luau < Formula
+  desc "Fast, safe, gradually typed embeddable scripting language derived from Lua"
+  homepage "https://luau-lang.org"
+  url "https://github.com/Roblox/luau/archive/0.504.tar.gz"
+  sha256 "544b3107e1c53f8349b1e327efed55ac29cc20a2983dd23031ab07c5b6944633"
+  license "MIT"
+  revision 0
+  head "https://github.com/Roblox/luau.git", branch: "master"
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+    system "cmake", "--build", ".", "--target", "Luau.Repl.CLI", "--config", "RelWithDebInfo"
+    system "cmake", "--build", ".", "--target", "Luau.Analyze.CLI", "--config", "RelWithDebInfo"
+    bin.install "luau", "luau-analyze"
```

Comment:
Can't cmake install this?

Reply:
There are no install rules defined in `CMakeLists` file from what I can see. I'm not that familiar with `cmake` so I may be missing something or misunderstanding though.

Reply:
CMake typically generates them for you if you don't specify custom ones.